src/BinSkim.Rules/RulesExtensionMethods.cs

```diff
@@ -92,7 +92,7 @@ public static string CreateOutputCoalescedByLibrary(this IList<ObjectModuleDetai
                 sb.Append(key);
 
                 List<string> objectModules = librariesToObjectModulesMap[key];
-                objectModules.Sort();
+                objectModules.Sort((a, b) => Path.GetFileNameWithoutExtension(a).CompareTo(Path.GetFileNameWithoutExtension(b)));
                 string[] oms = objectModules.ToArray();
```

Comment:
Same as above.

Reply:
Please comment this code, it's a non-obvious change. Did you find any online documentation (or PR) on this change?

Reply:
Behavior changes when comparing strings in .NET 5 and later (official reprint) (iditect.com)
https://iditect.com/article/behavior-changes-when-comparing-strings-in-net-5-and-later-official-reprint.html

Add the following item in the project file to revert back using NLS.
image